.travis.yml

```diff
@@ -17,17 +19,22 @@ before_install:
   - "curl http://localhost:5984/"  # print couch info
   - "uname -a"
   - "lsb_release -a"
+addons:
+  apt:
+    packages:
+    - moreutils
+    - libblas-dev
+    - liblapack-dev
 install:
-  - "sudo apt-get install moreutils libblas-dev liblapack-dev"
   - "git clone https://github.com/dimagi/commcarehq-venv.git"
   - "cp -r commcarehq-venv/hq_env/* ~/virtualenv/"
   - "source ~/virtualenv/bin/activate"
-  - "time (bash -e scripts/uninstall-requirements.sh | ts)"
```

Comment:
the new travis doesn't have the `ts` command. not sure if it's useful to time without the ts since travis already time each instruction